.github/CODEOWNERS

```diff
@@ -147,9 +147,9 @@ app/mailers/transactional_email_mailer.rb @department-of-veterans-affairs/my-edu
 app/mailers/veteran_readiness_employment_mailer.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/mailers/views/ch31_submissions_report.html.erb @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/mailers/views/dependents_application_failure.erb @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/mailers/views/direct_deposit.html.erb @department-of-veterans-affairs/vfs-authenticated-experience
-app/mailers/views/failed_claims_report.erb @department-of-veterans-affairs/vfs-authenticated-experience
-app/mailers/views/hca_submission_failure.html.erb @department-of-veterans-affairs/vfs-authenticated-experience
+app/mailers/views/direct_deposit.html.erb @department-of-veterans-affairs/vfs-authenticated-experience-backend
+app/mailers/views/failed_claims_report.erb @department-of-veterans-affairs/vfs-authenticated-experience-backend
+app/mailers/views/hca_submission_failure.html.erb @department-of-veterans-affairs/vfs-authenticated-experience-backend
```

Comment:
Are you keeping this file?

Reply:
No, but I was planning on reassigning code ownership in subsequent PRs.  Is this an issue?

Reply:
No, just wanted to verify that you weren’t planning on changing the file assignments here, just the team name